src/main/java/Command/DeleteCommand.java

```diff
@@ -0,0 +1,56 @@
+package Command;
+import Duke.DukeException;
+import Duke.DukeUi;
+import Duke.Storage;
+import Duke.Task;
+import Duke.TaskList;
+import java.io.IOException;
+
+/**
+* Command that deletes a task from the TaskList when executed.
+*/
+public class DeleteCommand extends Command {
+    boolean exit;
+    private String userAction;
+
+    public DeleteCommand(String userAction) {
+        this.exit = false;
+        this.userAction = userAction;
+    }
+
+    public boolean isNumeric(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
```

Comment:
Perhaps you may consider to put it in `execute` via `catch (NumberFormatException e)`